### PR TITLE
Update the version section for VS2017

### PR DESCRIPTION
**Pull request by Michael-Eng**
In VS2017, the version ranges specified for VS must use zero as the minor version. Updating the docs to reference that.

(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@Michael-Eng : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

File: docs/extensibility/vsix-extension-schema-2-0-reference.md [hunk 1]
```diff
@@ -124,7 +124,11 @@ A VSIX deployment manifest file describes the contents of a VSIX package. The fi
         -   Single version # - only the specified version.  
   
         > [!IMPORTANT]
-        >  Version 2.0 of the VSIX Schema was introduced in Visual Studio 2012. To use this schema you must have Visual Studio 2012 or later installed on the machine and use the VSIXInstaller.exe that is part of that product. You can target earlier versions of Visual Studio with a Visual Studio 2012 or later VSIXInstaller, but only by using the later versions of the installer.  
+        >  Version 2.0 of the VSIX Schema was introduced in Visual Studio 2012. To use this schema you must have Visual Studio 2012 or later installed on the machine and use the VSIXInstaller.exe that is part of that product. You can target earlier versions of Visual Studio with a Visual Studio 2012 or later VSIXInstaller, but only by using the later versions of the installer. 
+        
+        VS2017 version numbers can be found here: https://docs.microsoft.com/en-us/visualstudio/install/visual-studio-build-numbers-and-release-dates?view=vs-2017#feedback
```
Comment (gewarren) [inline docs/extensibility/vsix-extension-schema-2-0-reference.md]:
```suggestion
        Visual Studio 2017 version numbers can be found at [Visual Studio build numbers and release dates](/visualstudio/install/visual-studio-build-numbers-and-release-dates).
```

File: docs/extensibility/vsix-extension-schema-2-0-reference.md [hunk 2]
```diff
@@ -124,7 +124,11 @@ A VSIX deployment manifest file describes the contents of a VSIX package. The fi
...
+        
+        When expressing the for VS2017 releases, the minor version should always be zero. For example, if specifying VS2017 15.3.26730.0 as a minor version, it should be expressed as [15.0.26730.0,16.0). Note this is only true for VS2017 version numbers.
```
Comment (gewarren) [inline docs/extensibility/vsix-extension-schema-2-0-reference.md]:
```suggestion
        When expressing the version for Visual Studio 2017 releases, the minor version should always be **0**. For example, Visual Studio 2017 version 15.3.26730.0 should be expressed as [15.0.26730.0,16.0). This is only required for Visual Studio 2017 version numbers.
```

**Comment by PRMerger6** [comment]
@gewarren : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@gewarren : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

File: docs/extensibility/vsix-extension-schema-2-0-reference.md [hunk 3]
```diff
@@ -124,7 +124,11 @@ A VSIX deployment manifest file describes the contents of a VSIX package. The fi
...
+        >  Version 2.0 of the VSIX Schema was introduced in Visual Studio 2012. To use this schema you must have Visual Studio 2012 or later installed on the machine and use the VSIXInstaller.exe that is part of that product. You can target earlier versions of Visual Studio with a Visual Studio 2012 or later VSIXInstaller, but only by using the later versions of the installer. 
+        
+        Visual Studio 2017 version numbers can be found at [Visual Studio build numbers and release dates](/visualstudio/install/visual-studio-build-numbers-and-release-dates).
+        
+        When expressing the version for Visual Studio 2017 releases, the minor version should always be **0**. For example, Visual Studio 2017 version 15.3.26730.0 should be expressed as [15.0.26730.0,16.0). This is only required for Visual Studio 2017 version numbers.
```
Comment (gewarren) [inline docs/extensibility/vsix-extension-schema-2-0-reference.md]: BTW what is the ,16.0 for? And I also just noticed that the opening and closing parenthesis around the version number don't match.

File: docs/extensibility/vsix-extension-schema-2-0-reference.md [hunk 4]
```diff
@@ -124,7 +124,11 @@ A VSIX deployment manifest file describes the contents of a VSIX package. The fi
         -   Single version # - only the specified version.  
   
         > [!IMPORTANT]
-        >  Version 2.0 of the VSIX Schema was introduced in Visual Studio 2012. To use this schema you must have Visual Studio 2012 or later installed on the machine and use the VSIXInstaller.exe that is part of that product. You can target earlier versions of Visual Studio with a Visual Studio 2012 or later VSIXInstaller, but only by using the later versions of the installer.  
+        >  Version 2.0 of the VSIX Schema was introduced in Visual Studio 2012. To use this schema you must have Visual Studio 2012 or later installed on the machine and use the VSIXInstaller.exe that is part of that product. You can target earlier versions of Visual Studio with a Visual Studio 2012 or later VSIXInstaller, but only by using the later versions of the installer. 
+        
+        Visual Studio 2017 version numbers can be found at [Visual Studio build numbers and release dates](/visualstudio/install/visual-studio-build-numbers-and-release-dates).
```
Comment (gewarren) [inline docs/extensibility/vsix-extension-schema-2-0-reference.md]:
```suggestion
        Visual Studio 2017 version numbers can be found at [Visual Studio build numbers and release dates](../install/visual-studio-build-numbers-and-release-dates.md).
```

File: docs/extensibility/vsix-extension-schema-2-0-reference.md (hunk 3 above)
Comment (nschonni) [reply docs/extensibility/vsix-extension-schema-2-0-reference.md]: `,16.0)` will be 2019 (or whatever the name ends up being) and above

**Comment by gregvanl** [comment]
Please commit Genevieve's last relative path suggestion before you merge this. Thanks
#sign-off

**Comment by PRMerger7** [comment]
@GitHubber17 : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

**Comment by GitHubber17** [comment]
@gregvanl @gewarren - Per your request, I adjusted the URL to use the relative path.

Is this PR ready for merge? Thanks

**Comment by gregvanl** [comment]
@GitHubber17 Yes, you can merge the PR.

**Comment by Michael-Eng** [comment]
Thanks for the quick response everyone!

**Pull request merged**